tests/internal/test_settings.py

```diff
@@ -605,3 +606,22 @@ def test_remoteconfig_header_tags(run_python_code_in_subprocess):
         env=env,
     )
     assert status == 0, f"err={err.decode('utf-8')} out={out.decode('utf-8')}"
+
+
+def test_tracer_reconfigure_does_not_crash_tracer(run_python_code_in_subprocess):
+    env = os.environ.copy()
+    env.update({"DD_TRACER_PARTIAL_FLUSH_ENABLED": "true", "DD_TRACER_PARTIAL_FLUSH_MIN_SPANS": "1"})
```

Comment:
_probably_ don't need the env vars?

doing `.configure(partial_flush_enabled=True, partial_flush_min_spans=1)` _should_` be enough?

but all sort of same/same, so this isn't a blocking comment, feel free to resolve.

Reply:
🤔 , yeah I guess not, it's the "incoming" tracer configuration that needs to have partial flush enabled.